changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## NEXT
+
+- Modified `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.add_edge` in
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` to always order source and destination nodes by
+  `time_lag` even when the edge is not directed. For example, `add_edge('c', 'a (lag=1)', edge_type='--')` will add a
```

Comment:
Use enumeration for edge type. Don't want to recommend that the user uses string.